includes/EventManager.php

```diff
@@ -92,13 +93,15 @@ public function rest_api_init() {
 	/**
 	 * Add the weekly option to cron schedules if it doesn't exist
 	 *
-	 * @param  array  $schedules  List of cron schedule options
+	 * @param  array $schedules  List of cron schedule options
 	 *
 	 * @return array
 	 */
 	public function add_minutely_schedule( $schedules ) {
-		if ( ! array_key_exists( 'minutely',
-				$schedules ) || MINUTE_IN_SECONDS !== $schedules['minutely']['interval'] ) {
+		if ( ! array_key_exists(
+			'minutely',
+			$schedules
+		) || MINUTE_IN_SECONDS !== $schedules['minutely']['interval'] ) {
```

Comment:
```suggestion
		if ( 
		    ! array_key_exists( 'minutely', $schedules ) || 
		    MINUTE_IN_SECONDS !== $schedules['minutely']['interval'] 
		) {
```
Let's reformat this a bit differently.

Reply:
I did that initially, but it appears the PHP formatter has other plans. Let me see what works best.